.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,16 +7,23 @@ assignees: ""
 ---
 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. E.g. "I'm always frustrated when [...]"
+What problem are you trying to solve? E.g. "I'd really like to do X, but I can't because Y."
 
 **Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+What new thing would you like ReSpec to do? Is there a similar solution in other software?
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+**Describe any alternatives you've considered**
+How are you currently working around this limitation?
 
 **Can you or your organization fund the work or help with development?**
-ReSpec is a community maintained project but we don't work for free. To add a new feature, funding pays for the feature development, testing, documentation, server costs, etc. We have an open collective where development costs are tracked and shared. Please see our [Open Collective](https://opencollective.com/respec/).
+ReSpec is a community maintained project, meaning it doesn't have any dedicated staff or funding.
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+There are two options to get your feature implemented. You can:
+
+1. contribute to the project yourself via a pull request. We will [guide](https://respec.org/docs/#developers-guide) you!
+2. fund a contributor to implement your feature (US$500+ depending on complexity).
```

Comment:
And "just wait"? Maybe instead of "get your featured implemented", say prioritized?